bagit.py

```diff
@@ -632,6 +645,25 @@ def _validate_bagittxt(self):
             if first_line.startswith(codecs.BOM_UTF8):
                 raise BagValidationError("bagit.txt must not contain a byte-order mark")
 
+    def _path_is_dangerous(self, path):
+        """
+        Return true if path looks dangerous, i.e. potentially operates
+        outside the bagging directory structure, e.g. ~/.bashrc, ../../../secrets.json,
+            \\?\c:\, D:\sys32\cmd.exe
+        """
+        if os.path.isabs(path):
+            return True
+        if os.path.expanduser(path) != path:
+            return True
+        if os.path.expandvars(path) != path:
+            return True
+        real_path = os.path.realpath(os.path.join(self.path, path))
+        real_path = os.path.normcase(os.path.normpath(real_path))
```

Comment:
I feel like I had a reason for including it, but I can't imagine what it was. Removed!